taskcluster/config.yml

```diff
@@ -18,46 +18,48 @@ taskgraph:
 # The list of valid stages that can be used with `target-stage and `start-stage`.
 # These get attached to tasks in `kinds`.
 valid-stages:
-    - clean-corpus
-    - clean-mono
+    - alignments-backtranslations
+    - alignments-distillation
+    - alignments-teacher
+    - all-pipeline
+    - all-pr-pipeline
+    - analyze-corpus
+    - analyze-mono
     - bicleaner
     - bicleaner-model
-    - merge-corpus
-    - merge-devset
-    - merge-mono
-    - train-vocab
-    - train-backwards
+    - clean-mono
+    - clean-parallel
+    - collect-backtranslations
+    - collect-distillation-mono
+    - collect-distillation-parallel
+    - distillation-parallel-keep-best
+    - distillation-parallel-score
+    - distillation-parallel-shortlist
+    - distillation-parallel-src-extract-best
```

Comment:
Why is it called "distillation-parallel-src-extract-best"? It compares the translation to the reference, so it uses both source and target parts of the parallel corpus.